init.lua

```diff
@@ -1,3 +1,29 @@
+-- standard compatibility switcher block.
+
+local moditems = {}  -- switcher
+local mineclone_path = core.get_modpath("mcl_core") and mcl_core
+local default_path = core.get_modpath("default")
+
+if mineclone_path then -- means MineClone 2 is loaded, this is its core mod
+	moditems.iron_item = "mcl_core:iron_ingot"   -- MCL version of iron ingot
+	moditems.coal_item = "mcl_core:coalblock" -- MCL version of coal block
+	moditems.green_dye = "mcl_dye:green" -- MCL version of green dye
+	moditems.sounds = mcl_sounds.node_sound_defaults
+	moditems.infobox_can = {}
+	moditems.infobox_dump = {}
```

Comment:
This will work for a wrong reason. Table cannot be converted to a string. Please set this to `nil` to remove the metadata field.

Reply:
Ops, took me several minutes to find really silly mistake in the old code, when I had little idea of Lua (that commit from year ago). Ty.